subprojects/core/src/main/java/org/gradle/api/internal/DependencyClassPathProvider.java

```diff
@@ -53,17 +49,18 @@ public DependencyClassPathProvider(ModuleRegistry moduleRegistry, PluginModuleRe
     }
 
     @Override
+    @SuppressWarnings("deprecation")
```

Comment:
Why does this have a deprecation warning?

Reply:
It's from before I merged the `DependencyFactoryInternal`, I must have missed deleting it during the merge.  It's useless now, I'll remove it.